app/src/main/res/values-sv/strings.xml

```diff
@@ -308,85 +308,85 @@
     <string name="status_wallet_connect_wrongversion">Node version incompatible - please upgrade!</string>
 
     <string name="menu_info">Detaljer</string><!--Changed to: Show Secrets!-->
-    <string name="menu_streetmode">Street Mode</string>
+    <string name="menu_streetmode">Gatu Mode</string>
 
-    <string name="info_nodes_enabled">Node-o-matiC enabled, tap for more info.</string>
-    <string name="node_height">Last block updated: %1$s</string>
-    <string name="label_nodes">Nodes</string>
-    <string name="node_name_hint">Node Name (Optional)</string>
-    <string name="node_address_hint">Hostname</string>
+    <string name="info_nodes_enabled">Node-o-matiC aktiverad, tryck för mer info.</string>
+    <string name="node_height">Senaste blocket uppdaterat: %1$s</string>
+    <string name="label_nodes">Noder</string>
+    <string name="node_name_hint">Node Namn (Frivillig)</string>
+    <string name="node_address_hint">Värdnamn</string>
     <string name="node_port_hint">Port</string>
-    <string name="node_user_hint">Username (Optional)</string>
-    <string name="node_pass_hint">Password (Optional)</string>
-    <string name="node_host_unresolved">Cannot resolve host</string>
-    <string name="node_host_empty">We need this!</string>
-    <string name="node_port_numeric">Must be numeric</string>
-    <string name="node_port_range">Must be 1&#8211;65535</string>
-    <string name="node_fab_add">Add Node</string>
-    <string name="node_refresh_hint">Touch to refresh!</string>
-    <string name="node_test_error">CONNECTION ERROR %1$d</string>
-    <string name="node_general_error">CONNECTION ERROR</string>
-    <string name="node_auth_error">AUTHENTICATION FAILED</string>
-    <string name="node_result_label">Test Result:</string>
+    <string name="node_user_hint">Användare (Frivillig)</string>
+    <string name="node_pass_hint">Lösenord (Frivillig)</string>
+    <string name="node_host_unresolved">Det går inte att lösa värden</string>
+    <string name="node_host_empty">Vi behöver det här!</string>
+    <string name="node_port_numeric"Måste vara numeriskt</string>
```

Comment:
You forgot > there, it must be 
```<string name="node_port_numeric">Måste vara numeriskt</string>```
Then build will pass

Reply:
ok made the change, thanks for the help!